apps/paymaster-proxy/docker-compose.base.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
```

Comment:
i think it would be better to stick to a single `docker-compose` file and if you need duplication you can use variables like what i'm doing in the gateway repo, extends is really a relic before yaml variables were a thing